server/request-handlers/DynamicRequestHandler.ts

```diff
@@ -74,32 +68,40 @@ export default class DynamicRequestHandler extends RequestHandler {
             return;
         }
         if (this.res.statusCode !== 200) {
-            this.renderError(this.res.statusCode);
-            return;
+            return await this.renderError(this.res.statusCode);
         }
-        if (this.actionItem.useView) { // basically, if @eta.mvc.raw() is not set
+        if (result === undefined) { // undefined return means render the view
```

Comment:
Understand the comment explains the line, but is there any way to do this a little more self-explanatory rather than 'trust me this is what it does'